README.md

```diff
@@ -1,12 +1,14 @@
 # vscode-docs-authoring
 
-This repo contains code for the Docs Markdown VS Code extension, which provides Markdown authoring assistance for contributing to docs.microsoft.com. The Docs Markdown extension ships in the VS Code Marketplace, either as an individual extension or bundled with other useful extensions as part of the Docs Authoring Pack.
+This repo contains code for the Docs Markdown VS Code extension, which provides Markdown authoring assistance for contributing to docs.microsoft.com. The Docs Markdown extension ships in the VS Code Marketplace, either as an [individual extension](https://marketplace.visualstudio.com/items?itemName=docsmsft.docs-markdown) or bundled with other useful extensions as part of the [Docs Authoring Pack](https://marketplace.visualstudio.com/items?itemName=docsmsft.docs-authoring-pack).
 
-To accurately insert relative links, images, and other embedded content, you must have your VS Code workspace scoped to the root of your cloned OPS repo.
+The Docs Authoring Pack also includes [markdownlint](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint), a popular linter by David Anson, and [DocFx](https://marketplace.visualstudio.com/items?itemName=ms-docfx.DocFX) (beta), which enables a docs.microsoft.com-specific Markdown preview.
```

Comment:
Hi @meganbradley , just a quick question.  Is the DocFx extension considered a beta release?